tools/edta/edta_macros.xml

```diff
@@ -0,0 +1,70 @@
+<macros>
+    <xml name="bio_tools">
+        <xrefs>
+            <xref type="bio.tools">edta</xref>
+        </xrefs>
+    </xml>
+    <xml name="edta_main_param">
+        <!-- Genome to TE annotations -->
+        <param name="genome" argument="--genome" type="data" format="fasta" label="Genome File" help="The Genome FASTA file"/>
+        <param name='species' argument ="--species" type="select" label="Which species should be used for identification of TIR candidates">
+            <option value="Others">Others</option>
+            <option value="Rice">Rice</option>
+            <option value="Maize">Maize</option>
+        </param>
+        <param name="step" argument="--step" type="select" label="Specify which steps you want to run EDTA" help="all: run the entire pipeline (default) \n filter: start from raw TEs to the end.">
+            <option value="all">All</option>
+            <option value="filter">Filter</option>
+            <option value="final">Final</option>
+            <option value="anno">Anno</option>
+        </param>
+        <param name="cds" argument="--cds" type="data" format="fasta" label="Coding Sequence File" optional='true' help="Provide a FASTA file containing the coding sequence (no introns, UTRs, nor TEs) of this genome or its close relative."/> 
+        <param name="curatedlib" argument="--curatedlib" type="data" format="fasta" label="Curated Library File" optional='true' help="Provided a curated library to keep consistant naming and classification for known TEs. All TEs in this file will be trusted 100%, so please ONLY provide MANUALLY CURATED ones here."/>
+        <param name="sensitive" argument="--sensitive" type="boolean"  truevalue="1" falsevalue="0"  label="Use RepeatModeler to identify remaining TEs" help="Use RepeatModeler to identify remaining TEs This step is very slow and MAY help to recover some TEs."/>
```

Comment:
when you use "argument" you do not need to use name in this case as the "name" is generated out of argument

Also a nice trick is to use `truevalue="--sensitive 1"` this also simplifies a lot your command section

Reply:
ping